setup.py

```diff
@@ -2,23 +2,29 @@
 from setuptools import setup
 from setuptools import find_packages
 
+LONG_DESCRIPTION = ""
```

Comment:
why do you need to set this empty string? Won't line 7 stand alone?

Reply:
Great point, I was following form with the other `open` shortly after this one. I'll remove this and the other similar instantiation for the REQUIRED_PKGS to simplify.